tests/dummy/app/templates/identity/global-navigation.hbs

```diff
@@ -46,109 +54,117 @@
 				<ul>
 					<li>
-						When user provides at least 3 characters, show the list of suggestions by putting them in
-						<code class="format-code">.wds-global-navigation__search-suggestions</code> element as a list
+						When user provides at least 3 characters, show the list of suggestions by
+						putting them in
+						<code class="format-code">.wds-global-navigation__search-suggestions</code>
+						element as a list
 						and adding <code class="format-code">wds-is-active</code> class to the
-						<code class="format-code">.wds-global-navigation__search-input-wrapper</code> element.
+						<code
+							class="format-code">.wds-global-navigation__search-input-wrapper</code>
+						element.
 					</li>
 					<li>
-						When user presses up or down arrow keys, highlight the currently chosen one by adding
-						<code class="format-code">wds-is-selected</code> class to the proper <code class="format-code">li</code> element.
+						When user presses up or down arrow keys, highlight the currently chosen one
+						by adding
+						<code class="format-code">wds-is-selected</code> class to the proper <code
+						class="format-code">li</code> element.
 					</li>
 					<li>
-						When user presses the enter key while a suggestion is highlighted, go directly to the selected page.
+						When user presses the enter key while a suggestion is highlighted, go
+						directly to the selected page.
 					</li>
 				</ul>
 			</li>
 			<li>
 				Anon and logged in state
 				<ul>
 					<li>
-						For logged in users on tablet devices "Start a wiki" button changes into "+" icon.
+						For logged in users on tablet devices "Start a wiki" button changes into "+"
+						icon.
 						In order to make that happen <code class="format-code">.wds-global-navigation__start-a-wiki</code>
 						container requires additional class name: <code class="format-code">wds-user-is-logged-in</code>.
 					</li>
-					<li>For anon users render <code class="format-code">.wds-global-navigation__account-menu</code> div that contains user login and register buttons.</li>
-					<li>For logged in users render <code class="format-code">.wds-global-navigation__user-menu</code> div that contains user menu.</li>
+					<li>For anon users render <code class="format-code">.wds-global-navigation__account-menu</code>
+						div that contains user login and register buttons.
+					</li>
+					<li>For logged in users render <code class="format-code">.wds-global-navigation__user-menu</code>
+						div that contains user menu.
+					</li>
 				</ul>
 			</li>
 		</ul>
 	</section>
 
 	<section>
-		<h2 id="logged-out-with-hubs">Logged out with hubs</h2>
+		<h2 id="logged-out-with-hubs">Default logged out</h2>
 
-		{{#component-demo name='global-nav' codeOnly=true componentName='wds-global-navigation' standalone=true variantUrl='/identity/global-navigation/standalone/logged-out-with-hubs'}}
+		{{#component-demo name='global-nav' codeOnly=true componentName='wds-global-navigation'
```

Comment:
unfortunatelly this will break the demo
{{#component-demo have to be on one line :(

https://github.com/Wikia/design-system/blob/master/tests/dummy/app/components/component-demo.js#L5